src/providers/slack.ts

```diff
@@ -36,7 +37,21 @@ export class Slack implements OAuth2Provider {
 		});
 		const tokens: SlackTokens = {
 			accessToken: result.access_token,
-			idToken: result.id_token
+			idToken: result.id_token,
+			refreshToken: result.refresh_token ?? null,
+			accessTokenExpiresAt: typeof result.expires_in !== "undefined" && createDate(new TimeSpan(result.expires_in, "s")) || undefined
```

Comment:
This line is quite strange, but I found that `expires_in` field didn't appeared when I didn't use token rotation...

Reply:
I think that's expected since without token exchange, the access token last indefinitely. Should be `null` instead of `undefined` tho

Reply:
Thanks. That makes sense.
If I understood correctly, I'll change like this.
```suggestion
			accessTokenExpiresAt: typeof result.expires_in !== "undefined" && createDate(new TimeSpan(result.expires_in, "s")) || null
```

And in this case, should I also change the `accessTokenExpiresAt` field in `Token` type too? Since it doesn't accept null type.
https://github.com/pilcrowOnPaper/arctic/blob/69afa38a6f0c2def99a3c0f8c03e13478f901f92/src/index.ts#L107

Reply:
Didn't realize that! Yeah the type declaration should be updated to allow null

Reply:
Thanks! I re-request review😄